pkg/rbac/authzMiddleware.go

```diff
@@ -8,6 +8,11 @@ import (
 
 func AuthorizeUser(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// if postgres db is not setup, return error
```

Comment:
Maybe this should be a separate middleware.  It's a small check, but not required for authorization.